cudax/include/cuda/experimental/__execution/when_all.cuh

```diff
@@ -220,15 +217,12 @@ struct _CCCL_TYPE_VISIBILITY_DEFAULT when_all_t
         }
         else
         {
-          _CUDAX_TRY( //
-            ({ //
-              __errors_.template __emplace<_CUDA_VSTD::decay_t<_Error>>(static_cast<_Error&&>(__err));
-            }),
-            _CUDAX_CATCH(...) //
-            ({ //
-              __errors_.template __emplace<::std::exception_ptr>(::std::current_exception());
-            }) //
-          )
+          _CCCL_TRY({ //
+            __errors_.template __emplace<_CUDA_VSTD::decay_t<_Error>>(static_cast<_Error&&>(__err));
+          })
+          _CCCL_CATCH((...) { //
+            __errors_.template __emplace<::std::exception_ptr>(::std::current_exception());
+          })
```

Comment:
In a perfect world this would just be:
```suggestion
          _CCCL_TRY()
          {
            __errors_.template __emplace<_CUDA_VSTD::decay_t<_Error>>(static_cast<_Error&&>(__err));
          }
          _CCCL_CATCH(...)
          {
            __errors_.template __emplace<::std::exception_ptr>(::std::current_exception());
          }
```